test/e2e/framework/kcp.go

```diff
@@ -791,11 +791,11 @@ func unreadyComponentsFromError(err error) string {
 	return strings.Join(unreadyComponents, ", ")
 }
 
-// loadKubeConfig loads a kubeconfig from disk. This method is
+// LoadKubeConfig loads a kubeconfig from disk. This method is
 // intended to be common between fixture for servers whose lifecycle
 // is test-managed and fixture for servers whose lifecycle is managed
 // separately from a test run.
-func loadKubeConfig(kubeconfigPath string) (clientcmd.ClientConfig, error) {
+func LoadKubeConfig(kubeconfigPath, contextName string) (clientcmd.ClientConfig, error) {
```

Comment:
why do we need this? We shouldn't build a helper library for kube.

Reply:
I wanted to avoid copying of code that loads a kubeconfig

Reply:
xref: https://github.com/kcp-dev/kcp/pull/2620/files#diff-7fb917a8d5aa0dbb0fde1476d13f4692688a85b0d33691d858b20cae95457d3eR727

Reply:
@sttts k8s libraries have just a little too much power, every project I've ever seen has their own simplification to not copy-paste ...